aapcs64/aapcs64.rst

```diff
@@ -3111,6 +3112,65 @@ instead.
 The SVE tuple types are mangled using their ``arm_sve.h`` names
 (``svBASExN_t``).
 
+Function types which have SME streaming, ZA interface or ZT0 interface attributes must include these attributes in the name mangling of the type.
```

Comment:
Like @efriedma-quic says, the intention was to include this mangling if and only if the mangling would normally include the return type of the function.  We should probably say that explicitly.